app/questionnaire/placeholder_transforms.py

```diff
@@ -223,29 +206,6 @@ def format_date_range(self, date_range: tuple[str, str]) -> str:
 
         return f"{start_date_formatted} to {end_date_formatted}"
 
-    @staticmethod
-    def parse_date(date: str) -> datetime:
```

Comment:
Moved this to a top level function in the utils file. Could be a static method on operation but type hints and overloading does not work with static methods. https://github.com/python/mypy/issues/7781